ci/data.sh

```diff
@@ -27,12 +27,24 @@ for filename in "${files[@]}"; do
     entry_name=$(basename "${filename}" .md)
     entry_tag=${entry_name%_*}
     entry_date=${entry_name#*_}
-    printf "\055 semver: %s\n  date: %s\n  packager: \"\"\n  changes:" \
+
+    CHANGELOG_START="\055 semver: %s
+  date: %s
+  packager: Nordsec Ltd. <contact@nordteams.com>
```

Comment:
@keliramu Do you know if we have other contact info?

Reply:
@Savolro maybe knows

Reply:
I've changed to use `NordVPN Linux Team <linux@nordvpn.com>` instead